_posts/2016-03-28-Mutual-Problems.md

```diff
@@ -0,0 +1,130 @@
+---
+layout: post
+name: mutual-problems
+title: Mutual Problems
+date: 2016-03-28 00:00:00 +00:00
+author: Robert Firek
+image:
+    src: /assets/img/custom/blog/2016-03-28-https.jpg
+    attribution:
+        text: Image by Sean MacEntee (CC BY 2.0)
+        href: https://www.flickr.com/photos/smemon/15944989872/in/photostream/
+tags:
+- Java
+- HTTPS
+- Mutual Authentication
+--- 
+
+The HTTPS protocol is the well-established standard for securing our internet connections. Understanding how this protocol works is not problem and the corresponding [RFC document](https://tools.ietf.org/html/rfc2818) is available since 2000.
```

Comment:
I'd rather say `for securing our web connections` . Internet includes other non-HTTP services such as IMAP, FTP etc.